### PR TITLE
ThemeProvider fix for Table view

### PR DESCRIPTION
**Pull request by DanishMehmood-bit**
> Fixed ThemeProvider for Table view not inheriting the theme correctly.
> Moved iTwinui-react as a peer dep.

### CODE REVIEW

File: packages/modules/imodel-browser/package.json [hunk 1]
```diff
@@ -84,6 +78,10 @@
     "typescript": "^4.2.3"
   },
   "peerDependencies": {
+    "classnames": "^2.2.6",
+    "react-intersection-observer": "^8.31.1",
+    "@itwin/itwinui-icons-react": "^2.2.0",
+    "@itwin/itwinui-react": "^2.11.4",
```
[the remark this answers is not in the document]
Comment (DanishMehmood-bit) [reply packages/modules/imodel-browser/package.json:84]: No specific reasons, I can move them back to deps if you want. I only need itwinui-react and icons to be peers

[the remark this answers is not in the document]
Comment (ben-polinsky) [reply packages/modules/imodel-browser/package.json:84]: I think that's best unless there's a specific reason to do so. Thanks!

[the remark this answers is not in the document]
Comment (DanishMehmood-bit) [reply packages/modules/imodel-browser/package.json]: Done

**Review by ben-polinsky — APPROVED** [review]

**Review by raplemie — APPROVED** [review]
LGTM! Thanks!

**Pull request merged**